.github/workflows/release-c7-data-migrator.yml

```diff
@@ -149,4 +150,23 @@ jobs:
           mvn release:perform -B \
             -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}" \
             -DlocalCheckout=${{ inputs.IS_DRY_RUN }} \
-            -Darguments='-DskipTests=true -Dskip.central.release=${SKIP_REPO_DEPLOY} -Dskip.camunda.release=${SKIP_REPO_DEPLOY} -Dgpg.passphrase="${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}"'
+            -Psonatype-oss-release,distro \
+            -Darguments='-DskipTests=true -Dskip.central.release=true -Dskip.camunda.release=${SKIP_REPO_DEPLOY}'
```

Comment:
🙃  I am wondering if `SKIP_REPO_DEPLOY` name could be improved. Maybe `SKIP_RELEASE` or `SKIP_RELEASE_TO_NEXUS` & `SKIP_RELEASE_TO_MAVEN`.

Reply:
I would keep it for the time being as the code is inspired by the C8 repo. [link](https://github.com/camunda/camunda/blob/93d16bfb60ce843e988768d16b44eb73708829ac/.github/workflows/camunda-platform-release.yml#L152)